src/components/Dialogs/utils.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
```

Comment:
This file is mostly the same as `utils.ts`--just renamed as a `tsx` file with some stuff added

Reply:
this isnt totally true anymore lol. everything below line 50 is new